### DISCUSSION TITLE
Can't toggle menubar back on in v2.7.8; or Location of config files

### DISCUSSION THREAD

**Discussion by john-homenuke** [keepassxreboot/keepassxc#10756] [Q&A] [answered]
I'm running Ubuntu 22.04 and tried the new menubar toggle in v2.7.8. To my dismay, that toggle feature made itself vanish from the UI. I found the [pull request](https://github.com/keepassxreboot/keepassxc/pull/10341), which indicates the Alt key turns it back on, but my Alt key refuses to do that. I figure that is my own Ubuntu problem, although I couldn't find any shortcut overrides for my Alt key. So I went on the hunt for a config file that might contain the menubar toggle setting. Deleting `~/.cache/keepassxc/keepassxc.ini` had no impact.

Is there another config file out there somewhere that contains this setting? It's rather persistent.

**Comment by droidmonkey** [comment, marked as answer]
We also have configuration stored in ~/.config/keepassxc/keepassxc.ini

**Reply by john-homenuke** [reply]
Thank you. Much appreciated!
